view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -0,0 +1,178 @@
+        urlBuilder,
+        errorProcessor
+    ) {
+        'use strict';
+
+        return Component.extend({
+            defaults: {
+                template: 'Omise_Payment/payment/offsite-installment-form'
+            },
+
+            isPlaceOrderActionAllowed: ko.observable(quote.billingAddress() != null),
+
+            /**
+             * Get payment method code
+             *
+             * @return {string}
+             */
+            getCode: function () {
+                return 'omise_offsite_installment';
+            },
+
+            /**
+             * Initiate observable fields
+             *
+             * @return this
+             */
+            initObservable: function () {
+                this._super()
+                    .observe([
+                        'omiseOffsite',
+                        'installmentTerms'
+                    ]);
+
+                return this;
+            },
+
+            /**
+             * Get a checkout form data
+             *
+             * @return {Object}
+             */
+            getData: function () {
+                return {
+                    'method': this.item.method,
+                    'additional_data': {
+                        'offsite': this.omiseOffsite(),
+                        'terms': this.installmentTerms()
+                    }
+                };
+            },
+
+            /**
+             * Is method available to display
+             *
+             * @return {boolean}
+             */
+            isActive: function () {
+                return this.getOrderCurrency().toLowerCase() === 'thb' && this.getStoreCurrency().toLowerCase() === 'thb';
+            },
+
+            /**
+             * Returns Installment Terms
+             *
+             * @return {array}
+             */
+            getInstallmentTerms(id) {
+                const capabilities = checkoutConfig.payment.capabilities;
+                for (const key in capabilities) {
+                    if (capabilities[key]._id === 'installment_' + id)
+                        return ko.observableArray(capabilities[key].allowed_installment_terms);
+                }
+            },
+
+            /**
+             * Get order currency
+             *
+             * @return {string}
+             */
+            getOrderCurrency: function () {
+                return window.checkoutConfig.quoteData.quote_currency_code;
+            },
+
+            /**
+             * Get store currency
+             *
+             * @return {string}
+             */
+            getStoreCurrency: function () {
+                return window.checkoutConfig.quoteData.store_currency_code;
+            },
+
+            /**
+             * Check if order value meets minimum requirement
+             *
+             * @return {boolean}
+             */
+            orderValueTooLow: function () {
+                return window.checkoutConfig.totalsData.base_grand_total < 5000;
```

Comment:
This should be coming from capabilities API, or at least comment why it is hardcoded for now

Reply:
I described it in PR but maybe could add it into code..